### ISSUE TITLE
Quick Start broken

### ISSUE THREAD

**Issue by minguo33** [klesh/JigsawWM#8]
Hi, I have tried to get this working on Windows 11 and Windows 10, each time the experience was the same.

Installing jigsawwm from PIP and trying to use the quickstart example wm.pyw  nothing happens.  

When trying to call wm.pyw from a python interpreter receive this error:

```
wm.pyw, line 1, in <module>
    from jmk import hks
ImportError: cannot import name 'hks' from 'jmk' 

```
Trying to run jigsaw.pyw results again in nothing happening, and when calling it from a python interpreter receives this error:

```
jigsaw.pyw", line 2, in <module>
    from log import *
ModuleNotFoundError: No module named 'log'

```
Investigating that issue seems to imply python 'log' library has been renamed to 'logging' but I can't tell if that's what jigsawwm is trying use.

Interesting project, would like to test it out.
Are the example files still current?  If not, do you have a working example to use?

**Comment by klesh** [comment]
Thanks for reporting the problem, will look into it ASAP.

**Comment by klesh** [comment]
Hi, @minguo33 , Sorry for taking so long to investigate the issue. I think the problem is that you tried to import the `.pyw` file in the `examples` which is not what they are meant for.
Please copy the code from file you needed and create your own `pyw` file to launch the `jigsawwm`.

**Comment by minguo33** [comment]
> Hi, @minguo33 , Sorry for taking so long to investigate the issue. I think the problem is that you tried to import the `.pyw` file in the `examples` which is not what they are meant for. Please copy the code from the files you need and create your own `pyw` file to launch the `jigsawwm`.

Hi @klesh , thanks for taking a look.  Unfortunately, I haven't been able to get any of the examples to run whether or not I copy the code into a new file or download the source and try to run it.    Can you give some guidance on how to get a working configuration?

When trying to test import statements from my python interpreter, I can import jmk, but not hks.  (which is where my previous attempts got caught up.)  

```

>>> from jigsawwm import jmk
>>> from jigsawwm import hks
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ImportError: cannot import name 'hks' from 'jigsawwm' (\JigsawWM-main\src\jigsawwm\__init__.py)

```
My environment is python 3.12, jigsawwm was installed with pip, which self reports as version 2.0.

**Comment by klesh** [comment]
Hi, @minguo33 There is no `hks` module under the `jigsawwm`.
I think you copied the code from one of the examples, in this case, the `hks` is a variable from the `jmk.pyw` file which is an instance of a `jmk.Hotkeys` class for hotkeys subscription. Did you want to subscribe to hotkey events? if that so, you should creating your own instances just like the `jmk.pyw` does:
https://github.com/klesh/JigsawWM/blob/d54173aef0cad35f69a7ec76b0d962c45fd69d2c/examples/jmk.pyw#L153

**Comment by gazpachoking** [comment]
Looks like the wm.pwy example is wrong. Specifically this line:

https://github.com/klesh/JigsawWM/blob/c2756b898353f7df0966217693846c3a82fbec35/examples/wm.pyw#L1

**Comment by klesh** [comment]
@gazpachoking You are totally correct. I meant to show cases that you may do for different features and how to combine them together.
However, it does seem a little bit confusing, I'm open to suggestion of making it clearer.
